src/themes/teams/components/Divider/dividerVariables.ts

```diff
@@ -1,10 +1,11 @@
 export default (siteVars: any) => {
   return {
-    color: siteVars.gray02,
-    backgroundColor: siteVars.gray10,
-    typePrimaryColor: siteVars.brand,
-    typePrimaryBackgroundColor: siteVars.brand,
-    typeSecondaryColor: siteVars.gray04,
-    typeSecondaryBackgroundColor: siteVars.gray08,
+    dividerColor: siteVars.gray09,
+    textColor: siteVars.gray03,
```

Comment:
Please mention in the CHANGELOG that we are renaming this variables, so that the user can be aware if they were using them.